### PR TITLE
Feat/css and details cleanup

### LINKED ISSUE #21: Loading screen in case of slow api response

### PR DESCRIPTION
**Pull request by kirch1**
### Files Changed:
- most of them
### What Has Changed: clean up x
- clean up css for page parents, root, and app
- add tos library file
- add check boxes to select areas of concern
- logout button and user icon
- loading icon

### Closes: (any issues that it closes)
Closes #21

### CODE REVIEW

File: src/Components/App/App.tsx
Comment (Jesusborjas006) [inline src/Components/App/App.tsx:1]: Great job adding the logout functionality and removing code that is commented out.

File: src/Components/Process/Process.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      />
+    ));
+  };
+
+  return (
+    <main className="process-main">
+      <h1 className="heading">Terms of Service Processor</h1>
+      <p className="sub-heading">Understand what's important to you.</p>
+
+      <form className="form-card">
```
Comment (Jesusborjas006) [inline src/Components/Process/Process.tsx:64]: Love the loading animation when waiting for the results.

**Review by Jesusborjas006 — APPROVED** [review]

File: src/Components/App/App.tsx [hunk 2]
```diff
@@ -1,54 +1,65 @@
...
+                <Header logout={logout} user={user} />
+                <Results concerns={concerns} />
+              </>
+            ) : (
+              <Redirect to={'/process'}/>
+            )}
+          </Route>
+        </Switch>
       )}
-      {/* // <Header />
```
Comment (chrissycooper) [inline src/Components/App/App.tsx:40]: whoops forgot to take this out. THANK YOU

File: src/Components/Footer/Footer.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ const Footer = () => {
...
+      <p className='footer-text'>The information provided on this website is collected and analyzed by AI technology to help you understand how certain terms and conditions affect specific areas of concern, such as privacy or recurring payments. However, please be aware that the information is for general informational purposes only and is not intended to provide legal advice or create a lawyer-client relationship. The accuracy, completeness, and adequacy of the information is not guaranteed and you should consult with a licensed attorney in your jurisdiction for advice on specific legal issues or questions.</p>
```
Comment (chrissycooper) [inline src/Components/Footer/Footer.tsx:7]: It makes more sense to keep these classNames separate. Thank you so much for your attention to detail!

**Review by chrissycooper — APPROVED** [review]

**Pull request merged**